volatility3/framework/plugins/layerwriter.py

```diff
@@ -95,15 +100,16 @@ def _generator(self):
             if not self.config["layers"]:
                 self.config["layers"] = []
                 for name in self.context.layers:
-                    if not self.context.layers[name].metadata.get("mapped", False):
+                    if "mapped" not in self.context.layers[name].metadata:
```

Comment:
Not sure why this was changed, but I don't particularly have a problem with it.

Reply:
Yeah up to you. Just to avoid unnecessary code. **"x" not in**  already does the get("x", False), which not only the get but also the False are not needed.